Lib/test/pythoninfo.py

```diff
@@ -179,6 +179,9 @@ def collect_platform(info_add):
             info_add(f'platform.freedesktop_os_release[{key}]',
                      os_release[key])
 
+    if sys.platform == 'android':
+        info_add('platform.android_ver', platform.android_ver())
```

Comment:
I suggest call_func() to catch exceptions:

```suggestion
        call_func(info_add, 'platform.android_ver', platform, 'android_ver')
```